tasks/python-reference-activity/README.md

```diff
@@ -0,0 +1,32 @@
+# @cumulus/python-reference-activity
+
+This 'task' is a reference activity implementation that is included with Cumulus to allow integration testing of the `cumulus message adapter` and `cumulus-process-py` with a deployed activity.
+
+## Development
+
+Developmental use of this lambda is intended to be simple - the processing activity runs an activity through the CMA and returns a static processing output, integration tests can be then built against the `Reference Activity` step in the `PythonProcess` workflow in combination with the `Reference Task` step.
```

Comment:
Does it really need to be combined with the Reference Task? You could integration test it individually since it returns static outputs?

Reply:
What is the `PythonProcess` workflow?

Reply:
It doesn't *have* to be, no.      Given we're really testing single tasks, running it as a decoupled workflow seemed preferable, particularly given the remote configuration elements, but I'm happy to have that discussion

Reply:
My question isn't really around how the testing is implemented, more the documentation. This documentation implies that you have to use it with the Reference Task for testing, which doesn't seem true?

Reply:
Hmm, no that's not, for the most part, true.  There's an implicit test here in that there's a two step process with a remote message component, but that's not what this doc is talking about.

Reply:
```suggestion
Developmental use of this lambda is intended to be simple - the processing activity runs an activity through the CMA and returns a static processing output, integration tests can be then built against the `Reference Activity` step in the `PythonProcess` workflow. 
```